src/index.d.ts

```diff
@@ -25,6 +25,7 @@ declare module "react-contextmenu" {
         renderTag?: React.ElementType,
         mouseButton?: number,
         disableIfShiftIsPressed?: boolean,
+        [key: string]: any
```

Comment:
Also, `ContextMenuTriggerProps` don't seem to cater for the `contextTrigger` usage: `contextTrigger.handleContextClick(event)`?